src/templates/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    rules: {
+        'react/prop-types': 0,
```

Comment:
In order to enable this rule for other folders, please remove it from `/.eslintrc.js`.